pkg/linkeddata/linkeddata.go

```diff
@@ -47,3 +47,6 @@ type Proof struct {
 	Domain     *string   `json:"domain,omitempty"`
 	Nonce      *string   `json:"nonce,omitempty"`
 }
+
+// IsProof is used by gqlgen for the union type
+func (*Proof) IsProof() {}
```

Comment:
Is this supposed to do something when called?

Reply:
No, its some introspection thing that gqlgen does.  because the method exists its therefore implements the proof interface